src_js/components/sidebar/TableOfContents.tsx

```diff
@@ -40,6 +40,13 @@ export default function TableOfContents(props: PropsType) {
     };
 
     window.addEventListener('hashchange', hash_listener, false);
+
+    // Also account for user clicking the same sidebar item that is already selected [Fixes issue #66].
+    var sidebarTocListElements : any = document.body.querySelector('aside nav ul.primer-spec-toc-list');
```

Comment:
If we're going to add a click-handler, we might as well add it on the entire `<nav>` — in this case, we don't even *need* to use `useEffect()`! Move this logic to an `onClick` attribute of the `<nav>` in the `render()` method.

```jsx
<nav
  id="primer-spec-toc"
  onClick={() => {
    // Move the logic here
  }}
>
  {tocNodes}
</nav>
```